theano/gof/cmodule.py

```diff
@@ -1565,6 +1565,15 @@ def get_gcc_shared_library_arg():
     else:
         return '-shared'
 
+def get_icc_shared_library_arg():
+    """
+    Return the platform-dependent GCC argument for shared libraries.
```

Comment:
As this is a copy-paste of the gcc version, why not rename the gcc version to get_shared_library_arg? Do you expect that to change in the futur? If you think it is better to keep it separated, update the docstring.